pkg/datapath/linux/node_ids.go

```diff
@@ -77,8 +72,28 @@ func (n *linuxNodeHandler) GetNodeIP(nodeID uint16) string {
 	return n.nodeIPsByIDs[nodeID]
 }
 
+func (n *linuxNodeHandler) GetNodeID(nodeIP net.IP) (uint16, bool) {
```

Comment:
do we want to go with an `error` instead of the `bool` in the "public" facing API?

Reply:
I agree a bool will be fine as it mimics the map "interface" just fine